onnxruntime/core/providers/coreml/builders/impl/builder_utils.cc

```diff
@@ -80,18 +82,20 @@ common::Status HandleAutoPad(const std::vector<int64_t> input_shape,
   return Status::OK();
 }
 
+void CreateCoreMLWeight(CoreML::Specification::WeightParams& weight,
+                        const float* data, size_t num_elements) {
+  weight.mutable_floatvalue()->Clear();
+  std::copy(data, data + num_elements,
+            google::protobuf::RepeatedFieldBackInserter(weight.mutable_floatvalue()));
+}
```

Comment:
Is it possible to use Reserve and copy_n, or alternatively call Reserve prior to the copy? Not a big deal thought given this is during setup.

Reply:
Tried reserve and copy_n, seems not working (UT failure), added reserve before the copy

Reply:
Sorry - mean Resize + copy_n. But Reserve is fine too - just may not translate to a block copy but at least will only have one memory allocation.

Reply:
Switched to list initialization